WalletWasabi.Fluent/Behaviors/HistoryItemDetailsBehavior.cs

```diff
@@ -12,7 +12,7 @@ namespace WalletWasabi.Fluent.Behaviors
 {
 	public class HistoryItemDetailsBehavior : DisposingBehavior<DataBoxRow>
 	{
-		private HistoryItemDetailsAdorner? _historyItemDetailsAdorner;
+		private DataBoxItemDetailsAdorner? _historyItemDetailsAdorner;
```

Comment:
Shouldn't we also rename the variable `_historyItemDetailsAdorner`?

Reply:
i renamed it here, somehow it got lost in the aether haha